configure.ac

```diff
@@ -1591,21 +1606,13 @@ AS_IF([test x"$with_default_string" = "xunsafe"],
   [default_safe_string=false],
   [default_safe_string=true])
 
-oc_cflags="$common_cflags $internal_cflags"
-oc_cppflags="$common_cppflags $internal_cppflags"
-ocamlc_cflags="$common_cflags $sharedlib_cflags"
+OCAML_JOIN_FLAGS(oc_cflags,$common_cflags,$internal_cflags)
+OCAML_JOIN_FLAGS(oc_cppflags,$common_cppflags,$internal_cppflags)
+OCAML_JOIN_FLAGS(ocamlc_cflags,$common_cflags,$sharedlib_cflags)
```

Comment:
This change stopped trailing spaces appearing in `Makefile.config` - it's not mandatory, it just made diff'ing the files easier!